utils/manifests/utils.go

```diff
@@ -0,0 +1,229 @@
+package manifests
+
+import (
+	"bytes"
+	"encoding/json"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/layer5io/meshkit/models/oam/core/v1alpha1"
+)
+
+func getDefinitions(crd string, resource int, cfg Config, filepath string, binPath string) (string, error) {
+	var def v1alpha1.WorkloadDefinition
+	definitionRef := strings.ToLower(crd) + ".meshery.layer5.io"
+	apiVersion, err := getApiVersion(binPath, filepath, crd)
+	if err != nil {
+		return "", ErrGetAPIVersion(err)
+	}
+	apiGroup, err := getApiGrp(binPath, filepath, crd)
+	if err != nil {
+		return "", ErrGetAPIGroup(err)
+	}
+	//getting defintions for different native resources
+	def.Spec.DefinitionRef.Name = definitionRef
+	def.ObjectMeta.Name = crd
+	switch resource {
+	case SERVICE_MESH:
+		{
+			def.Spec.Metadata = map[string]string{
+				"@type":         "pattern.meshery.io/mesh/workload",
+				"meshVersion":   cfg.MeshVersion,
+				"meshName":      cfg.Name,
+				"k8sAPIVersion": apiGroup + "/" + apiVersion,
+				"k8skind":       crd,
+			}
+		}
+	case K8s:
+		{
+			def.Spec.Metadata = map[string]string{
+				"@type":         "pattern.meshery.io/k8s",
+				"k8sAPIVersion": apiGroup + "/" + apiVersion,
+				"k8skind":       "",
```

Comment:
This is supposed to be `Kind` of resource. 
Example: `Service`, `Endpoint`, etc.

Reply:
Refer: https://github.com/meshery/meshery/blob/master/oam/workloads/service.k8s_definition.json